src/main/archetype/ui.frontend/webpack.common.js

```diff
@@ -89,7 +90,10 @@ module.exports = {
             }),
             new TSLintPlugin({
                 files: ['./**/components/**/*.ts', './**/components/**/*.tsx']
-            })
+            }),
+            new CopyPlugin([
+              { context: './src/main/webpack/', from: 'resources/**/*.*', to: './clientlib-site/', force:true },
```

Comment:
I think this can be simplified to avoid the use of a hard coded path and I'd avoid using force=true...

```
{ from: path.resolve(__dirname, SOURCE_ROOT + '/resources'), to: './clientlib-site/resources' }
```